x-pack/packages/ml/aiops_utils/src/components/progress_controls.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiProgress, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+interface ProgressControlProps {
+  progress: number;
+  progressMessage: string;
+  onRefresh: () => void;
+  onCancel: () => void;
+  isRunning: boolean;
+}
+
+export function ProgressControls({
+  progress,
+  progressMessage,
+  onRefresh,
+  onCancel,
+  isRunning,
+}: ProgressControlProps) {
+  return (
+    <EuiFlexGroup>
+      <EuiFlexItem>
+        <EuiFlexGroup direction="column" gutterSize="none">
+          <EuiFlexItem data-test-subj="aiopProgressTitle">
+            <EuiText size="xs" color="subdued">
+              <FormattedMessage
+                data-test-subj="aiopsProgressTitleMessage"
+                id="xpack.aiops.progressTitle"
+                defaultMessage="Progress: {progress}% — {progressMessage}"
+                values={{ progress: Math.round(progress * 100), progressMessage }}
+              />
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiProgress
```

Comment:
A bad choice of data set, but with the `event_rate` data set, the analysis stops at 20%. As discussed, it should abort and return 100%

![image](https://user-images.githubusercontent.com/7405507/176143065-6f4d8656-d23e-4bca-8c99-458728e9a8e2.png)

Reply:
Fixed in [9c3ad32](https://github.com/elastic/kibana/pull/135058/commits/9c3ad3212b05c42fc98339134c6a92a2c3fb76de).